test_scripts/template_of_script.lua

```diff
@@ -28,39 +31,31 @@ local commonFunctions = require ('user_modules/shared_testcases/commonFunctions'
 --[[ @Example: the function gets....
 --! @parameters:
 --! func_param ]]
-local function Example(func_param)
+local function preconditions()
   -- body
 end
 
---[[ General Precondition before ATF start ]]
--- General precondition for restoring configuration files of SDL:
-commonFunctions:SDLForceStop()
-
---[[ General Settings for configuration ]]
--- if not applicable remove this section
--- This part is under clarification, based on section for using common functions
-Test = require('connecttest')
-require('user_modules/AppTypes')
-
---[[ Preconditions ]]
--- if not applicable remove this section
-commonFunctions:newTestCasesGroup("Preconditions")
-function Test:Precondition_DESCRIPTION()
+local function postconditions()
   -- body
 end
 
---[[ Test ]]
-commonFunctions:newTestCasesGroup("Test")
--- Each Test will be separate and defined as one or few TestSteps
-function Test:TestStep_DESCRIPTION()
+local function positiveScenario()
   -- body
 end
 
---[[ Postconditions ]]
--- if not applicable remove this section
-commonFunctions:newTestCasesGroup("Postconditions")
-function Test:Postcondition_DESCRIPTION()
+local function negativeScenario()
   -- body
 end
 
-return Test
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment, start SDL, HMI", preconditions)
+
+runner.Title("Test")
+runner.Step("Positive scenario", positiveScenario)
+runner.Step("Negative scenario", negativeScenario)
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", postconditions)
+
+
```

Comment:
@dboltovskyi  Please remove redundant whitespaces.

Reply:
@aderiabin Corrected in 02b6871